doc/source/whatsnew/v0.18.1.txt

```diff
@@ -22,6 +22,8 @@ New features
 Enhancements
 ~~~~~~~~~~~~
 
+Partial string matches on `DateTimeIndex` now work when part of a `MultiIndex`
```

Comment:
use double-back ticks around ``DatetimeIndex`` (sp as well) (and same around ``MultiIndex``); put the issue reference on the same line